network/src/errors/network.rs

```diff
@@ -63,6 +63,22 @@ impl NetworkError {
             _ => false,
         }
     }
+
+    pub fn is_trivial(&self) -> bool {
```

Comment:
there's some overlap with `is_fatal` here; maybe `is_connection_breaking` would be a more obvious name here? also, `BrokenPipe` would probably fit there too

Reply:
broken pipe would be an implementation issue/error, and i haven't seen any. on board to rename though.